src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1186,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes removes prefix from parameter fullString if prefix occurs at the start of the string.
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param fullString  The full string to remove prefix from
+     * @param prefix  The prefix to remove from fullString
+     * @return  fullString without the prefix.
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
     }
```

Comment:
Good job for changing the header comment and parameters.  However, it doesn't remove the sign occurs at the start of the string only , right?